docresources/VersionBot/versionbot.md

```diff
@@ -115,7 +113,94 @@ procbot:
     minimum_version: 0.5
 ```
 
-Other bots are free to add any properties they require. It should be noted that all derived bots are able to view the entire configuration file on request.
+
+#### Maintainer and Reviewer Configuration
+
+A list of valid maintainers and approved reviewers may be defined to allow VersionBot to vet those Github users who are providing review approval.
+
+The definition for these users in the configuration file is as follows:
+
+````
+minimum_approvals: <number>
+reviewers:
```

Comment:
Because it was missed previously. I'd rather add it here than not have it in at all.